app/controllers/works_controller.rb

```diff
@@ -26,7 +27,7 @@ def new
 
     @content = Content.create!(user: current_user)
     @work_form = new_work_form
-    @license_presenter = LicensePresenter.new(work_form: @work_form, collection: @collection)
+    set_license_presenter
```

Comment:
Ended up doing a little refactoring to get the complexity down on `update`.

Reply:
@justinlittman What do you think about adding a comment here remind us that we can't move this line to the `before_action` above because we need to set `@collection` and `@work_form` to special values in `#new` before that method is called?

Reply:
Makes sense. Fixing.